howso/utilities/utilities.py

```diff
@@ -1173,7 +1173,7 @@ def deep_update(base, updates):
 def matrix_processing( # noqa
     matrix: pd.DataFrame,
     normalize: bool = False,
-    normalize_method: Iterable[str | Callable, str, Callable] = "relative",
+    normalize_method: Iterable[t.Literal["relative", "fractional", "feature_count"] | Callable] | t.Literal["relative", "fractional", "feature_count"] | Callable = "relative",
```

Comment:
Please also update the docstring for this parameter to include the literals.

Reply:
And, we shouldn't have the `Union`s in the docstring either. The docstring can be more human readable, so, instead of `Union` or `|`, just say "or".